traitsui/qt4/ui_panel.py

```diff
@@ -596,6 +596,21 @@ def __init__(self, group, ui, suppress_label=False):
                 # Create an editor.
                 self._setup_editor(group, GroupEditor(control=outer))
 
+            if group.scrollable:
+                # ensure we have a widget rather than a layout for the scroll area
```

Comment:
Nit: Local flake8 🤖  Line too long

Reply:
Need to look into my VS Code setup - it didn't flag this 😞